components/PlantDetails.js

```diff
@@ -69,9 +69,21 @@ export default function PlantDetails({ favoriteIDs, onToggleFavorite }) {
         <StyledListElement>
           Growing Conditions: {plant.growingConditions}
         </StyledListElement>
-        <StyledListElement>Water demand: {plant.waterDemand}</StyledListElement>
         <StyledListElement>
-          Nutrient demand: {plant.nutrientDemand}
+          Water demand:{" "}
+          {plant.waterDemand === "1"
+            ? "Low"
+            : plant.nutrientDemand === "2"
+            ? "Medium"
+            : "High"}
```

Comment:
im not sure but isnt there an easier way ? maybe a switch case function?

Reply:
Talked to Philip about this suggestion and he mentioned that a switch is not possible in return / jsx. Therefore we first need to extract the function and call it in the return afterwards. As this is currently just a work around and might be changed while focusing on styling and optic, we would therefore prefer to keep it like this for the moment.